src/MathCaptcha.php

```diff
@@ -65,8 +79,24 @@ public function reset()
     {
         $this->session->forget('mathcaptcha.first');
         $this->session->forget('mathcaptcha.second');
+        $this->session->forget('mathcaptcha.operand');
     }
 
+    /**
+     * Operand to be used ('*','-','+')
+     * 
+     * @return character
+     */
+    protected function getMathOperand()
+    {
+        if (!$this->session->get('mathcaptcha.operand')) {
+            $this->session->put('mathcaptcha.operand', $this->operandsArray[rand(0, 2)]);
```

Comment:
Just some tips: This will fail once you change the `this->operandsArray`. There is a helper function in Laravel `Arr::random()` or even go old-fashioned `$this->operandsArray[rand(0, count($this->operandsArray) - 1)]`.